### PR TITLE
Remove the Cluster level keys info

### PR DESCRIPTION
**Pull request by Sakpal**
Preview: https://colin.preview.docs.r3.com/en/platform/corda/5.2/deploying-operating/keys.html

### CODE REVIEW

File: content/en/platform/corda/5.2/deploying-operating/keys/_index.md [hunk 1]
```diff
@@ -214,7 +214,8 @@ To rotate the master wrapping key, do the following:
    {{% /tab %}}
    {{< /tabs >}}
 
-   Corda will wrap any new managed wrapping keys with this new master wrapping key.
+   The system will take some time to propagate newly added master key across Corda. Corda will wrap any new managed wrapping keys with this new master wrapping key.
```
Comment (nadinequinr3) [inline content/en/platform/corda/5.2/deploying-operating/keys/_index.md:217]:
```suggestion
   Corda will take some time to propagate the newly added master key across the cluster. Corda will wrap any new managed wrapping keys with this new master wrapping key.
```

File: content/en/platform/corda/5.2/deploying-operating/keys/_index.md [hunk 2]
```diff
@@ -247,38 +248,35 @@ To rotate the master wrapping key, do the following:
 
 ### Rotating Managed Wrapping Keys
 
-The cluster and virtual node wrapping keys are managed by Corda and can be rotated using the POST method of the [/api/v5_2/wrappingkey/rotation/{tenantid} endpoint](../reference/rest-api/openapi.html#tag/Key-Rotation-API/operation/post_wrappingkey_rotation__tenantid_). Specify one of the following as the path parameter:
+Virtual node wrapping keys are managed by Corda and can be rotated using the POST method of the [/api/v5_2/wrappingkey/rotation/{tenantid} endpoint](../reference/rest-api/openapi.html#tag/Key-Rotation-API/operation/post_wrappingkey_rotation__tenantid_). Specify the following as the path parameter:
```
Comment (nadinequinr3) [inline content/en/platform/corda/5.2/deploying-operating/keys/_index.md:251]:
```suggestion
Virtual node wrapping keys are managed by Corda and can be rotated using the POST method of the [/api/v5_2/wrappingkey/rotation/{tenantid} endpoint](../reference/rest-api/openapi.html#tag/Key-Rotation-API/operation/post_wrappingkey_rotation__tenantid_). Specify the short hash holding ID of the virtual node as the path parameter. For example:
```

File: content/en/platform/corda/5.2/deploying-operating/keys/_index.md [hunk 3]
```diff
@@ -247,38 +248,35 @@ To rotate the master wrapping key, do the following:
...
 
 * The short hash holding ID of the virtual node to rotate a virtual node wrapping key.
```
Comment (nadinequinr3) [inline content/en/platform/corda/5.2/deploying-operating/keys/_index.md:253]:
```suggestion
```

File: content/en/platform/corda/5.2/deploying-operating/keys/_index.md [hunk 4]
```diff
@@ -247,38 +248,35 @@ To rotate the master wrapping key, do the following:
...
-* One of the following for the corresponding cluster-level service:
-  * `rest`
-  * `crypto`
 
-For example, to rotate the cluster REST wrapping key:
+For example, to rotate the Virtual Node wrapping key:
```
Comment (nadinequinr3) [inline content/en/platform/corda/5.2/deploying-operating/keys/_index.md:255]:
```suggestion
```

**Comment by anien** [comment]
Looks good once we also commit Nadine's requested changes.

**Review by nadinequinr3 — APPROVED** [review]

**Pull request merged**